### ISSUE TITLE
Monaco intermittently scrolls down a line when selecting text in Edge 

### ISSUE THREAD

**Issue by jayspadie** [microsoft/monaco-editor#629]
monaco-editor version: 0.10.1
Browser: Edge (40.15063.0.0)
OS: Windows 10

This is a minor annoyance that was originally reported as a bug in Visual Studio Team Services, but appears to be a general issue with Monaco when using Edge.  When carefully dragging to select a longer line of text, Monaco will intermittently scroll down a line (sometimes two) causing an additional line of text to be selected.  It will repro about half the time for me, and after the initial scroll of one or two lines, it often does not repro again on the same line.  This only occurs with Edge.

This can be reproduced in the Monaco playground.  For example, see the animated gif below recorded from https://microsoft.github.io/monaco-editor/playground.html#interacting-with-the-editor-adding-an-action-to-an-editor-instance.  In this case, it scrolled two lines on selection, though it typically only does one.  Following that, the same line of text could be selected without triggering the scroll.  This doesn't appear to be an issue with a jittery mouse hand, but a legitimate difference in Edge.

![monacoscrollsonselectioninedge](https://user-images.githubusercontent.com/5246869/32784989-5ca4a0f6-c91e-11e7-895a-722ceb13e4b8.gif)

**Comment by rebornix** [comment]
I didn't successfully reproduce this issue but what I ran into often is while selecting content from left to right, sometimes it selects from several lines above. After debugging this problem, I found we already have some code to guard that https://github.com/Microsoft/vscode/blob/master/src/vs/editor/browser/controller/mouseTarget.ts#L525 , which is a Edge bug, but seems the bug is still there. I'm wondering if it's the same root cause leading to this.
